airbyte-integrations/bases/source-acceptance-tests/src/main/java/io/airbyte/integrations/acceptance_tests/source/full_refresh/FullRefreshAcceptanceTest.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * MIT License
+ *
+ * Copyright (c) 2020 Airbyte
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package io.airbyte.integrations.acceptance_tests.source.full_refresh;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import io.airbyte.protocol.models.ConfiguredAirbyteCatalog;
+import org.junit.jupiter.api.Test;
+
+public abstract class FullRefreshAcceptanceTest {
```

Comment:
What's the reason to use an abstract class pattern with the test runner extensions instead of an interface? I don't see usages of the base class and the presence of a base class isn't apparent to me. Are we setting this up for future PRs?